Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=9334e90d5ac5ee1fa6d8b75acb7c64a8907787d1
Commit:     9334e90d5ac5ee1fa6d8b75acb7c64a8907787d1
Parent:     1dd3294677d5484e4813c6c454ba7f192a3ef61d
Author:     Jiri Kosina <jkosina@suse.cz>
AuthorDate: Tue Jan 6 01:35:45 2009 -0800
Committer:  Dmitry Torokhov <dmitry.torokhov@gmail.com>
CommitDate: Tue Jan 6 01:38:52 2009 -0800

    Input: i8042 - add Dell Vostro 1510 to nomux list
    
    Dell Vostro needs 'nomux' quirk, otherwise the touchpad misbehaves.
    
    Reported-by: Robert Kiwanuka <robert.kiwanuka@gmail.com>
    Signed-off-by: Jiri Kosina <jkosina@suse.cz>
    Signed-off-by: Dmitry Torokhov <dtor@mail.ru>
---
 drivers/input/serio/i8042-x86ia64io.h |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/drivers/input/serio/i8042-x86ia64io.h b/drivers/input/serio/i8042-x86ia64io.h
index e196bf2..6fa2def 100644
--- a/drivers/input/serio/i8042-x86ia64io.h
+++ b/drivers/input/serio/i8042-x86ia64io.h
@@ -359,6 +359,13 @@ static struct dmi_system_id __initdata i8042_dmi_nomux_table[] = {
 			DMI_MATCH(DMI_PRODUCT_NAME, "HEL80I"),
 		},
 	},
+	{
+		.ident = "Dell Vostro 1510",
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Dell Inc."),
+			DMI_MATCH(DMI_PRODUCT_NAME, "Vostro1510"),
+		},
+	},
 	{ }
 };
 
--
